src/main/java/com/hri/hri_web_backend/controller/PublicationController.java

```diff
@@ -0,0 +1,66 @@
+package com.hri.hri_web_backend.controller;
+
+import javax.validation.Valid;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.hri.hri_web_backend.domain.PublicationType;
+import com.hri.hri_web_backend.dto.PublicationDto;
+import com.hri.hri_web_backend.global.StatusEnum;
+import com.hri.hri_web_backend.global.SuccessResponse;
+import com.hri.hri_web_backend.service.PublicationService;
+
+import lombok.RequiredArgsConstructor;
+
+@RestController
+@RequiredArgsConstructor
+public class PublicationController {
+
+	private final PublicationService ps;
+
+	//ResponseBody
+	//조회
+	@GetMapping(value = "/publications/{type}")
+	public SuccessResponse getPublicationByType(@PathVariable PublicationType type){
+		return SuccessResponse.builder()
+			.status(StatusEnum.OK)
+			.message("Publication 조회 성공")
+			.data(ps.findPublicationsByType(type))
+			.build();
```

Comment:
반환되는 성공응답 내에서 메소드를 호출하는 것 보단 `getPublicationByType`메소드에서 호출하는 형태로 수정하는 것이 어떨까요?
```suggestion
            List<Publication> publications = publicationService.findPublicationsByType(type);
            return SuccessResponse.builder()
		            .status(StatusEnum.OK)
		            .message("Publication 조회 성공")
		            .data(publications)
		            .build();
```
이렇게 구분지으면 해당 메소드의 의미가 더 명확하게 보일 것 같아 제안드립니다!

Reply:
확실히 그러네요 👍  수정하겠습니다!